Squirrel/Program.cs

```diff
@@ -76,6 +77,17 @@
     app.UseSwaggerUI();
     app.UseDeveloperExceptionPage();
 }
+app.UseCors(builder => builder
```

Comment:
Can be simplified with AllowAnyOrigin, because I don't see any regularity between these urls

Reply:
AllowAnyOrigin does not work with WithCredentials